www/src/states/use-chat-state.ts

```diff
@@ -187,6 +224,13 @@ const useChatStoreBase = create<ChatStore>()(
     {
       name: "chat-storage",
       getStorage: () => localStorage,
+      migrate: (persistedState: any, version: number) => {
```

Comment:
😮 zustand ❤️  cc @devjoaov you'll enjoy this!

have you been able to test this @luizmuraro ? I can help with that

Reply:
I've tested it going to the main branch, creating some messages there and then, after coming back here, everything was migrated! But would be great to make some other tests!